### ISSUE TITLE
Generated junit XML report currently missing required property 'classname' for each testcase

### ISSUE THREAD

**Issue by heimwege** [ArnaudBuchholz/ui5-test-runner#88]
Hey hey 👋🏻 

according to the [junit spec](https://llg.cubic.org/docs/junit/) a property `classname` for each `testcase` is 'required' (e,g, containing the value of the `package` property of the testsuite aka. the journey name). This is currently not part of the xml file generated by `$/junit-xml-report.js`

actual:

```
<testsuites>
    <testsuite
        name="http://localhost:8080/test/opaTests.qunit.html"
        package="Foo Journey"
        tests="1"
      >
        <testcase
              name="Test Bar" 
              time="3.374"
        >
        </testcase>
    </testsuite>
</testsuites>
```

expected:

```
<testsuites>
    <testsuite
        name="http://localhost:8080/test/opaTests.qunit.html"
        package="Foo Journey"
        tests="1"
      >
        <testcase
              name="Test Bar" 
              time="3.374"
              classname="Foo Journey"
        >
        </testcase>
    </testsuite>
</testsuites>
```

**Comment by ArnaudBuchholz** [comment]
Good catch, thx for reporting. Will be added in next release.

**Comment by ArnaudBuchholz** [comment]
Fixed in `5.0.0`
